src/Renderer.php

```diff
@@ -82,6 +84,7 @@ public function registerFunctions()
         $this->engine->registerFunction('file', [$this, 'file']);
         $this->engine->registerFunction('inline', [$this, 'inline']);
         $this->engine->registerFunction('inlineJson', [$this, 'inlineJson']);
+        $this->engine->registerFunction('tableOfContents', [$this, 'tableOfContents']);
```

Comment:
`getTableOfContents()`? I know the others aren't named like that but the rest are more utility-like and the next few functions all are verbs so we're not being super consistent through the class anyway 🤷🏻‍♀️